CONTRIBUTING.md

```diff
@@ -12,27 +12,30 @@ elsewhere.
 
 * Code and comments should be formatted to a width no greater than 120 columns
 * Files should be exempt of trailing spaces
-* Each abstraction should live in its own Scala file, i.e `RequestReader.scala`
-* Each implementation should live in the package object, i.e, `io.finch.request`
+* Each abstraction with corresponding implementations should live in its own Scala file, i.e `RequestReader.scala`
+* Each implicit conversion (if possible) should be defined in the corresponding companion object
 
 That said, the Scala style checker `sbt scalastyle` should pass on the code. 
 
 ## Write Tests
-Finch uses [ScalaTest][2] with the following settings:
+Finch uses both [ScalaTest][2] and [ScalaCheck][3] with the following settings:
 
-* Every test should be a `FlatSpec` with `Matchers` mixed in
+* Every test should be a `FlatSpec` with `Matchers` and `Checkers` mixed in
 * An assertion should be done with `x shouldBe y`
```

Comment:
`check` expects its argument to return a boolean (or something similar—not `Unit`, in any case), which means that assertions in properties have to be written with `===`.